Assets/Scripts/Utility/BuildingNames.cs

```diff
@@ -145,6 +145,8 @@ public static string GetName(int seed, DFLocation.BuildingTypes type, int factio
                                 break;
                             }
                         }
+                        if (a[a.Length - 1] == '.')
```

Comment:
Could you instead use `a = a.TrimEnd(".")` here? That might look a little neater and be friendlier to string class. Result will be same string if does not in fullstop.

Reply:
sure ;)